web/js/modules/layers/selectors.js

```diff
@@ -202,27 +246,16 @@ export function getLayers(layers, spec = {}, state) {
 }
 
 function forGroup(group, spec = {}, activeLayers, state) {
-  const projId = spec.proj || state.proj.id;
+  const projId = state.proj.id;
   let results = [];
   const defs = lodashFilter(activeLayers, { group });
   lodashEach(defs, (def) => {
-    // Skip if this layer isn't available for the selected projection
-    if (!def.projections[projId] && projId !== 'all') {
-      return;
-    }
-    if (
-      spec.dynamic
-      && !['subdaily', 'daily', 'monthly', 'yearly'].includes(def.period)
-    ) {
-      return;
-    }
-    if (
-      spec.renderable
-      && !isRenderable(def.id, activeLayers, spec.date, state)
-    ) {
-      return;
-    }
-    if (spec.visible && !def.visible) {
+    const notInProj = !def.projections[projId];
+    const dynamic = spec.dynamic
```

Comment:
I know this wasn't really your PR specific, but I believe this `spec.dynamic` isn't being used and can be removed.

Reply:
Nice, I'm all for cleanup if it's near my changes!